core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -31,6 +32,19 @@ class UserServiceSpec extends FlatSpecLike with TestDriverComponent with Mockito
   val defaultMockGcsDAO: GoogleServicesDAO = new MockGoogleServicesDAO("test")
   val testConf: Config = ConfigFactory.load()
 
+  // A workspace with the which namespaceName equals to defaultBillingProject's billing project name.
+  val workspace = Workspace(
+    defaultBillingProjectName.value,
+    testData.wsName.name,
+    UUID.randomUUID().toString,
+    "aBucket",
+    Some("workflow-collection"),
+    currentTime(),
+    currentTime(),
+    "test",
+    Map.empty
```

Comment:
I think this is only required in `fail with a 400 when workspace exists in this billing project to be deleted`, define it in that function, not at the class level

Reply:
Done